API/Services/Interfaces/IPlayerService.cs

```diff
@@ -1,22 +1,14 @@
 using API.DTOs;
-using API.Osu.Enums;
 
 namespace API.Services.Interfaces;
 
 public interface IPlayerService
 {
-    /// <summary>
-    /// Denotes whether a player for the given id exists
-    /// </summary>
-    Task<bool> ExistsAsync(int id);
-
     Task<IEnumerable<PlayerDTO>> GetAllAsync();
+
```

Comment:
I think we only use blank lines in cases where there's documentation, unless this was done by the formatter?

Reply:
Formatter I think :P Either way, I'll be doing some code quality stuff on the repos soon that will include docs 👍